schema_registry.go

```diff
@@ -103,3 +109,30 @@ func CreateSchema(
 	}
 	return schemaInfo, nil
 }
+
+func GetSubjectName(schema string, topic string, element Element, subjectNameStrategy string) (string, *Xk6KafkaError) {
+	if subjectNameStrategy == "" || subjectNameStrategy == "TopicNameStrategy" {
+		return topic + "-" + string(element), nil
+	}
+
+	var schemaMap map[string]interface{}
+	err := json.Unmarshal([]byte(schema), &schemaMap)
+	if err != nil {
+		return "", NewXk6KafkaError(failedEncodeToAvro, "Schema invalid json", nil)
```

Comment:
The message and the error code should be fixed.
```suggestion
		return "", NewXk6KafkaError(???, "Failed to unmarshal schema", nil)
```